pkgs/applications/virtualization/libgovirt/default.nix

```
@@ -1,46 +1,46 @@
 { lib
 , stdenv
 , fetchurl
-, glib
 , gnome
-, librest
-, libsoup
+, meson
 , pkg-config
 , gobject-introspection
+, ninja
+, librest_1_0
 }:
 
 stdenv.mkDerivation rec {
   pname = "libgovirt";
-  version = "0.3.8";
+  version = "0.3.9";
 
   outputs = [ "out" "dev" ];
 
   src = fetchurl {
     url = "mirror://gnome/sources/libgovirt/${lib.versions.majorMinor version}/${pname}-${version}.tar.xz";
-    sha256 = "HckYYikXa9+p8l/Y+oLAoFi2pgwcyAfHUH7IqTwPHfg=";
+    sha256 = "sha256-sS/DV58tAH5D/vHcwPSz+MEHDC+2o/WLI8U8dmJDIXw=";
   };
 
-  patches = lib.optionals stdenv.isDarwin [
-    # The flag breaks the build on darwin and doesn't seem necessary
-    ./no-version-script-ld-flag.patch
+  # https://gitlab.gnome.org/GNOME/libgovirt/-/issues/9
+  patches = [
```

Comments should go next to patches so that when another patch is added, they do not get orphaned:

```suggestion
  patches = [
    # https://gitlab.gnome.org/GNOME/libgovirt/-/issues/9
```